app/util/k8s/dcapt.tfvars

```diff
@@ -110,7 +110,7 @@ jira_image_repository = "atlassian/jira-software"
 
 # Supported versions by DCAPT: https://github.com/atlassian/dc-app-performance-toolkit#supported-versions
 # Jira version
-jira_version_tag = "9.12.12"
+jira_version_tag = "9.12.17"
```

Comment:
We need to move to 10.3 as a default version is this the latest LTS release now.